testsuite/forge/src/backend/k8s/chaos.rs

```diff
@@ -26,6 +26,12 @@ macro_rules! BANDWIDTH_NETWORK_CHAOS_TEMPLATE {
     };
 }
 
+macro_rules! NETWORK_LOSS_CHAOS_TEMPLATE {
+    () => {
+        "chaos/network_loss.yaml"
```

Comment:
why this needs to be a macro?

Reply:
Following convention above :P

Reply:
that's a very .. weird convention

Reply:
Oh actually there is a reason, this apparently delays the load for include_str to runtime instead of compile time

https://github.com/aptos-labs/aptos-core/blob/6dc1c36df520e1e2fa7c999a24399a285daa3cde/testsuite/forge/src/backend/k8s/cluster_helper.rs#L62